.github/workflows/lint.yaml

```diff
@@ -16,6 +16,8 @@ jobs:
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
     steps:
+      - name: apt update
```

Comment:
Yes, it was failing because the `apt` images were out of date on the Actions images I think. This ensures that `apt` is updated before we try to get the packages we need.